### PR TITLE
add tests for null assessment scores

### PR DESCRIPTION
**Pull request by ljanda**
## Description & motivation
Adds tests for assessments and objective assessments that will throw a warning if there are any null scores

## Breaking changes introduced by this PR:
None since these are tests that only throw warnings

## PR Merge Priority:
- [x] Low

## New files created:
- `cfg_assessment_scores` : Throws a warning if assessment scores are null 
- `cfg_objective_assessment_scores` : Throws a warning if objective assessment scores are null 

## Tests and QC done:
ran compiled test in snowflake to check functionality

## edu_wh PR Review Checklist:
Make sure the following have been completed before approving this PR:
- [x] Description of changes has been added to Unreleased section of [CHANGELOG.md](/CHANGELOG.md). Add under `## New Features` for features, etc.

### CODE REVIEW

File: tests/config_tests/cfg_assessment_scores.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+with fct_student_assessment  as (
+    select * from analytics.prod_wh.fct_student_assessment
```
Comment (rlittle08) [inline tests/config_tests/cfg_assessment_scores.sql:2]: need to replace these hard coded database/schema with `{{ref}}` syntax

File: tests/config_tests/cfg_assessment_scores.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+),
+dim_assessment as (
+    select * from analytics.prod_wh.dim_assessment
+),
+xwalk_assessment_scores as (
+    select * from analytics.prod_seed.xwalk_assessment_scores
+),
+joined as (
+    select distinct
+        dim_assessment.assessment_identifier,
```
Comment (rlittle08) [inline tests/config_tests/cfg_assessment_scores.sql:12]: picky thing, please put tenant & year before assessment_identifier

File: tests/config_tests/cfg_objective_assessment_scores.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+),
+dim_objective_assessment as (
+    select * from {{ ref('dim_objective_assessment') }}
+),
+xwalk_objective_assessment_scores as (
+    select * from {{ ref('xwalk_objective_assessment_scores') }}
+),
+joined as (
+    select distinct
+        dim_objective_assessment.assessment_identifier,
```
Comment (rlittle08) [inline tests/config_tests/cfg_objective_assessment_scores.sql:19]: picky thing, please put tenant & year before assessment_identifier

File: tests/config_tests/cfg_assessment_scores.sql
Comment (rlittle08) [inline tests/config_tests/cfg_assessment_scores.sql:1]: please add a config block at the top (as you have in the other one)

**Review by rlittle08 — CHANGES_REQUESTED** [review]

**Review by rlittle08 — APPROVED** [review]
